### PR TITLE
Remove dhouse

### PR DESCRIPTION
**Pull request by MasterWayZ**

### CODE REVIEW

File: test/integration/bitbar/inspec/bitbar_devicepool_spec.rb [hunk 1]
```diff
@@ -8,13 +8,6 @@
     end
   end
 
-  describe user('dhouse') do
```
Comment (aerickson) [inline test/integration/bitbar/inspec/bitbar_devicepool_spec.rb:11]: I think we should replace with you vs delete. I guess slightly more involved than just changing the login here, so let's punt to another PR.

File: test/integration/linux/inspec/users_spec.rb [hunk 2]
```diff
@@ -5,16 +5,11 @@
   describe user('aerickson') do
     it { should exist }
   end
-
```
Comment (aerickson) [inline test/integration/linux/inspec/users_spec.rb:8]: Replace with you or another team member vs delete.

File: test/integration/bitbar/inspec/bitbar_devicepool_spec.rb (hunk 1 above)
Comment (MasterWayZ) [reply test/integration/bitbar/inspec/bitbar_devicepool_spec.rb:11]: Will make a new PR.

File: test/integration/linux/inspec/users_spec.rb (hunk 2 above)
Comment (MasterWayZ) [reply test/integration/linux/inspec/users_spec.rb:8]: Done.

File: test/integration/bitbar/inspec/bitbar_devicepool_spec.rb (hunk 1 above)
Comment (MasterWayZ) [reply test/integration/bitbar/inspec/bitbar_devicepool_spec.rb:11]: Made https://github.com/mozilla-platform-ops/ronin_puppet/pull/482

**Review by aerickson — APPROVED** [review]

**Pull request merged**